### ISSUE TITLE
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` 

### ISSUE THREAD

**Issue by milanamatya** [imandyie/react-native-airbnb-clone#5]
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:167:1306)
    at ResolutionRequest.resolveDependency (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:80:16)
    at DependencyGraph.resolveDependency (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\node-haste\DependencyGraph.js:237:485)
    at Object.resolve (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\lib\transformHelpers.js:116:25)
    at dependencies.map.result (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\DeltaBundler\traverseDependencies.js:298:29)
    at Array.map (<anonymous>)
    at resolveDependencies (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\DeltaBundler\traverseDependencies.js:294:16)
    at E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\DeltaBundler\traverseDependencies.js:159:33
    at Generator.next (<anonymous>)
    at step (E:\xampp_new\htdocs\react-native-airbnb-clone\node_modules\metro\src\DeltaBundler\traverseDependencies.js:239:307)
 BUNDLE  [android, dev] ./index.js ââââââââââââââââ 0.0% (1/52), failed.

**Comment by SuiCxDe007** [comment]
Same error here.
